gsoc/models.py

```diff
@@ -3,6 +3,23 @@
 from django.contrib.auth.models import User
 
 
+class Scheduler(models.Model):
+    commands = (
+        ('send_email', 'send_email'),
+        ('send_irc_msg', 'send_irc_msg')
+    )
+    
+    id = models.AutoField(primary_key=True)
+    command = models.CharField(name='command', max_length=20, choices=commands)
+    data = models.TextField(name='data')
+    success = models.BooleanField(name='success', null=True, default=None)
```

Comment:
The default value of BooleanField is None when Field.default isn’t defined so is the default really necessary?